edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs

```diff
@@ -114,8 +121,6 @@ public static async Task<int> MainAsync(IConfiguration configuration)
                 int idleTimeoutSecs = configuration.GetValue(Constants.IdleTimeoutSecs, 300);
                 TimeSpan idleTimeout = TimeSpan.FromSeconds(idleTimeoutSecs);
                 ExperimentalFeatures experimentalFeatures = ExperimentalFeatures.Create(configuration.GetSection("experimentalFeatures"), logger);
-                string iothubHostname;
```

Comment:
why move these 2 variables?

Reply:
they're used below to init metrics provider ```MetricsProvider("edgeagent", iothubHostname, deviceId)```